functions/cloud_functions/userCreated.js

```diff
@@ -0,0 +1,53 @@
+const functions = require('firebase-functions');
+const admin = require('firebase-admin')
+
+
+exports.userCreated = functions.auth.user().onCreate(async user => {
+
+    const db = admin.firestore()
+
+    const newUser = {
+        firstName : "",
+        lastName : "",
+        middleName : "",
+        phone : "",
+        kinFirstName : "",
+        kinLastName : "",
+        kinMiddleName : "",
+        kinPhone : "",
```

Comment:
What are these fields for?

Reply:
Next of Kin Information... It's gonna be one of the things they can update on their profile page.

Reply:
Ooh, thanks.